app/retail/templates/emails/funder_payout_reminder.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize static %}
+
+{% block content %}
+
+<h1>{% trans "Your funded bounty is fulfilled" %}</h1>
+
+
+<p style="color: #4A4A4A;">
+  {% if bounty_fulfillment.fulfiller_github_url %}
+  {% blocktrans %}Heads up! The funded issue that you posted (below) has work submitted by <a href="https://github.com/{{ github_username }}">@{{ github_username }}.</a>{% endblocktrans %}
+  <br>
+  <br>
+  <img src="{% static "v2/images/update-notification.png" %}" style="padding: 20px 0;">
+  <br>
+  <br>
```

Comment:
would it make sense to use margin-top and margin-bottom as opposed to using soo many breaks ?